doc/guide/cockpit-file.xml

```diff
@@ -14,11 +14,11 @@
     </para>
 <programlisting>
 file = cockpit.file(path,
-                    { syntax: syntax_object,
-                      binary: boolean,
-                      max_read_size: int,
-                      superuser: string,
-                      host: string
+                    { "syntax": syntax_object,
+                      "binary": boolean,
+                      "max-file-size": int,
```

Comment:
Agreed about the renaming. However, disagreed about the dashes. This would be the *only* place in the Cockpit where a dash is being used as a function argument/object property. We have several with underscores, like `full_name` in [users](https://cockpit-project.org/guide/latest/cockpit-login.html) or `exit_{status,signal}`  in [process  errors](https://cockpit-project.org/guide/latest/cockpit-spawn.html) or `archive_source` in [metrics](https://cockpit-project.org/guide/latest/cockpit-metrics.html). Our function names also use underscores; the only exception are error/problem codes, but they aren't object properties or function arguments, just a property value.

It also makes code harder to write as you have to quote the identifier. 

Can this please be changed back to an underscore?

Reply:
I understand the desire to not have to quote.

However, this is the only underscore in the protocol. All other fields in the protocol use dashes. See ```"exit-status"``` and "exit-signal". See https://github.com/cockpit-project/cockpit/blob/master/doc/protocol.md for more.

If you insist I will leave this as is. However it is the one and only set of underscores in the protocol that I can find. ```archive_source``` and ```full_name``` are not in the protocol, unless they have escaped my notice somehow :)

Reply:
I see, so here the protocol shines through the JS API. Apparently in all other cases it doesn't, either by accident (we only happen to expose single-word identifiers from  the protocol), or by rewriting the JS API function arguments into different protocol identifiers. Would the latter be an option here? If that's too ugly, then it's not too hard to live with this API oddity, but it does stick out like a sore thumb - after all, the API is the primary thing Cockpit devs interact with, not the wire protocol.

Reply:
Will do.